buster/utils.py

```diff
@@ -1,6 +1,18 @@
 import os
 import urllib.request
 import zipfile
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+class UserInputs:
```

Comment:
Not sure this is the best place to put it, but I don't have a better suggestion.

Reply:
i moved it inside the `completers` folder, i think it makes more sense there.